library/network/src/modules/NetworkInterfaces.rb

```diff
@@ -599,30 +600,22 @@ def CanonicalizeIP(ifcfg)
       ifcfg = deep_copy(ifcfg)
       return nil if ifcfg.nil?
 
-      ip_and_prefix = Builtins.splitstring(
-        Ops.get_string(ifcfg, "IPADDR", ""),
-        "/"
-      )
-      ipaddr = Ops.get(ip_and_prefix, 0, "")
-      return deep_copy(ifcfg) if ipaddr == "" # DHCP or inconsistent
+      ipaddr, prefixlen = ifcfg["IPADDR"].to_s.split("/")
```

Comment:
well, if you do
```(ifcfg["IPADDR"] || "/").to_s.split("/", -1)```
you can avoid those ```to_s``` further. It's on your decision if that is improvement.